packages/core/src/editor.module.css

```diff
@@ -37,7 +37,6 @@ Tippy popups that are appended to document.body directly
 .defaultStyles h2,
 .defaultStyles h3,
 .defaultStyles li {
-  all: unset !important;
```

Comment:
we can't remove this one, as it's used to make sure styles from the host page are not carried over into blocknote